src/matrix.jl

```diff
@@ -395,27 +395,25 @@ v = L(u, p, t) # == A * u + B * (p * t)
 ```
 
 """
-struct AffineOperator{T,AT,BT,bT,C,F,F!} <: AbstractSciMLOperator{T}
+struct AffineOperator{T,AT,BT,bT,F,F!} <: AbstractSciMLOperator{T}
```

Comment:
rewrote operator evaluation methods for affine op to eliminate the need for cache.